src/Header.js

```diff
@@ -0,0 +1,30 @@
+// The Header creates links that can be used to navigate
+// between routes.
+import Link from "react-router-dom/es/Link";
```

Comment:
Is the link a function that is exported?  As in found it would be `{ Link }` rather than just regular `Link`